lib/rest/rest.go

```diff
@@ -292,11 +292,39 @@ func (api *Client) Call(opts *Opts) (resp *http.Response, err error) {
 // contentName - the name of the parameter for the file
```

Comment:
Can you update the comment please as to what it returns?

I think the best name for it would be `overhead` as it is the overhead of the multpart encoding.